.travis.yml

```diff
@@ -1,7 +1,11 @@
 sudo: false
-cache: pip
+cache:
+  apt: true
+  directories:
+    - $HOME/.cache/pip
 
 addons:
+  postgresql: "9.5"
```

Comment:
why not 9.6?

Reply:
No particular reason, just taking the example of 10.0. I can change it to 9.6.